packages/iTwinUI-react/src/core/Table/cells/DefaultCell.tsx

```diff
@@ -6,9 +6,22 @@ import React from 'react';
 import { CellRendererProps } from 'react-table';
 import cx from 'classnames';
 
-export type DefaultCellProps<
-  T extends Record<string, unknown>
-> = CellRendererProps<T> & React.ComponentPropsWithoutRef<'div'>;
+export type DefaultCellProps<T extends Record<string, unknown>> = {
+  /**
+   * Custom icon to be displayed at the beginning of the cell.
+   */
+  startIcon?: JSX.Element;
+  /**
+   * Custom icon to be displayed at the end of the cell.
+   */
+  endIcon?: JSX.Element;
+  /**
+   * Status of the cell.
+   * @default ''
+   */
+  status?: 'positive' | 'negative' | 'warning';
+} & CellRendererProps<T> &
```

Comment:
hmm.. that means if user puts `EditableCell` it is not possible to have statuses or icons.. right?

Reply:
Yes, add status to EditableCell?

Reply:
Yeah, so not sure :/
Does it make sense to have status and edit its content? I guess initially we can leave it out, but if needed, I kinda dont like duplicating code over both cells. Just not sure if there is something better apart from just reusing the type.

Reply:
Status on editable cell could be useful in verification, eg. only numbers are allowed. But we could leave that for future.